src/hooks/useStatus.ts

```diff
@@ -0,0 +1,36 @@
+import { useQuery, QueryObserverResult } from 'react-query';
```

Comment:
I also don't know if I understand the pattern of these files, but it seems like this is how all of these hooks are defined so I'm sure there is a reason for it so its all good.